README.md

```diff
@@ -213,7 +213,7 @@ sudo /etc/init.d/incus restart
 ## Configuration
 Incus needs to be restarted after any configuration change.
 
-#### CLIENT_BROADCASTS
+#### ClientBroadcast
```

Comment:
This should be changed back as the configuration option should still be the underscore'd version. see config.go for more info.